main/inc/lib/sessionmanager.lib.php

```diff
@@ -9886,4 +9886,194 @@ private static function compareByCourse($listA, $listB)
+                    }
+
+                    if (!empty($coachId)) {
+                        $values['id_coach'] = $coachId;
+                    }
+
+                    if (!empty($sessionCategoryId)) {
+                        $values['session_category_id'] = $sessionCategoryId;
+                    }
+
+                    if (!empty($description)) {
+                        $values['description'] = $description;
+                    }
+
+                    if (!empty($showDescription)) {
+                        $values['show_description'] = $showDescription;
+                    }
+
+                    if (!empty($duration)) {
+                        $values['duration'] = $duration;
+                    }
+
+                    if (!empty($visibility)) {
+                        $values['visibility'] = $visibility;
+                    }
+
+                    if (!empty($promotionId)) {
+                        $values['promotion_id'] = $promotionId;
+                    }
+
+                    if (!empty($displayStartDate)) {
+                        $values['display_start_date'] = api_get_utc_datetime($displayStartDate);
+                    }
+
+                    if (!empty($displayEndDate)) {
+                        $values['display_end_date'] = api_get_utc_datetime($displayEndDate);
+                    }
+
+                    if (!empty($accessStartDate)) {
+                        $values['access_start_date'] = api_get_utc_datetime($accessStartDate);
+                    }
+
+                    if (!empty($accessEndDate)) {
+                        $values['access_end_date'] = api_get_utc_datetime($accessEndDate);
+                    }
+
+                    if (!empty($coachStartDate)) {
+                        $values['coach_access_start_date'] = api_get_utc_datetime($coachStartDate);
+                    }
+
+                    if (!empty($coachEndDate)) {
+                        $values['coach_access_end_date'] = api_get_utc_datetime($coachEndDate);
+                    }
+
+                    if (!empty($sendSubscriptionNotification)) {
+                        $values['send_subscription_notification'] = $sendSubscriptionNotification;
+                    }
+                }
+
+                Database::update(
+                    $tbl_session,
+                    $values,
+                    ['id = ?' => $id]
+                );
+
+                if (!empty($extraFields)) {
+                    $extraFields['item_id'] = $id;
+                    $sessionFieldValue = new ExtraFieldValue('session');
+                    $sessionFieldValue->saveFieldValues($extraFields);
+                }
+
+                return $id;
+            }
+        }
+    }
+
+    /**
+     * Get the session information by id.
+     *
+     * @param int $id
+     *
+     * @return mixed false if the session does not exist, array if the session exist
+     */
+    public static function get_session_by_id($id)
```

Comment:
Use camelCase for methods

Reply:
Aunque existen dos métodos que llegan a hacer lo mismo

https://github.com/chamilo/chamilo-lms/blob/5e4a6f40a93217d23643b85ebbe08f6d4e9fbffc/main/inc/lib/sessionmanager.lib.php#L49

https://github.com/chamilo/chamilo-lms/blob/5e4a6f40a93217d23643b85ebbe08f6d4e9fbffc/main/inc/lib/sessionmanager.lib.php#L7660